src/ExpressionEvaluator/Core/Source/ResultProvider/SystemHelpers.cs

```diff
@@ -0,0 +1,45 @@
+﻿// A few dependencies from System.dll, which we want to avoid when compiling against Desktop Framework 2.0.
+namespace System.Linq
```

Comment:
I would move this to the bottom and add a comment to the affect of "// Avoid use of Linq when compiling against newer versions of mscorlib (for portability)..."